### PR TITLE
add keystone domain by rule conf

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,2 +1,3 @@
-- Fix: avoid modify ruleName epl if already unique (#654)
-- Fix: preselect in ruleWithContext (#630)
+- Add: service to authentication action parameters
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: No associated issue?

File: bin/perseo [hunk 2]
```diff
@@ -105,7 +105,8 @@ function loadConfiguration() {
         'PERSEO_AUTHENTICATION_HOST',
         'PERSEO_AUTHENTICATION_PORT',
         'PERSEO_AUTHENTICATION_USER',
-        'PERSEO_AUTHENTICATION_PASSWORD'
+        'PERSEO_AUTHENTICATION_PASSWORD',
+        'PERSEO_AUTHENTICATION_SERVICE'
```
Comment (fgalan) [inline bin/perseo:109]: Should this new PERSEO_AUTHENTICATION_SERVICE be included in documentation?

Comment (AlvaroVega) [reply bin/perseo:109]: It's related with previous issue https://github.com/telefonicaid/perseo-fe/issues/638, it's and improvement

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (fgalan) [reply CHANGES_NEXT_RELEASE:1]: Confirmed F2F: no issue associated

NTC

Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Add: service to authentication action parameters (and associated env var PERSEO_AUTHENTICATION_SERVICE)
```

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**